Scripts/gen_Digitize.sh

```diff
@@ -137,13 +137,14 @@ then
   ln -s $DSTEPS ${PRIMARY}.txt
 else
   samListLocations ${SAMOPT} --defname="dts.mu2e.${PRIMARY}.${CAMPAIGN}${PRIMARY_VERSION}.art" > ${PRIMARY}.txt
+
 fi
 echo \#include \"Production/JobConfig/digitize/Digitize.fcl\" >> digitize.fcl
 echo \#include \"Production/JobConfig/digitize/${DIGITYPE}.fcl\" >> digitize.fcl
 # turn off streams according to the digitization type.
 DIGOUT=""
 if [[ "${DIGITYPE}" == "Extracted" || "${DIGITYPE}" == "NoField" ]]; then
-  DIGOUT=${PRIMARY}
+  DIGOUT=${PRIMARY}Digi # TODO need to understand why this is the same name as the primary, throws error as name is same as previous stage
```

Comment:
This shouldn't be necessary, as the primary has file family dts while digis are dig.  Including Digi in the description should be redundant.  Where is this error coming from?